.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Is this file intended to be committed? If so, could you add a one liner on its function for posterity

Reply:
Yes its intended to be committed. I will add a comment.

Reply:
Comment has been applied. Thanks